backend/src/extension.ts

```diff
@@ -147,8 +147,8 @@ export class YeomanUIPanel {
 		this.rpc = new RpcExtension(this.panel.webview);
 		const outputChannel: YouiLog = new OutputChannelLog();
 		const vscodeYouiEvents: YouiEvents = new VSCodeYouiEvents(this.rpc, this.panel);
-
-		this.yeomanui = new YeomanUI(this.rpc, vscodeYouiEvents, outputChannel, this.logger, YeomanUIPanel.genFilter);
+		const outputFolder = _.get(vscode, "vscode.workspace.workspaceFolders[0].uri.path");
```

Comment:
remove vscode before workspace (in the string)

Reply:
fixed